addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3361,6 +3362,20 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
      * @returns {Int} the maximum width at which the image can be displayed
      */
     _computeMaxDisplayWidth() {},
+    /**
+     * Adds background-image when image is optimizable.
+     *
+     * @private
+     * @abstract
+     */
+    _forceOptions(img) {},
+    /**
+     * @private
+     * @returns {Boolean}
+     */
+    _isImgOptimizable(img) {
```

Comment:
No need for this method anymore, just use the code inline: in stable, the less change the better.